gobbler-server.rkt

```diff
@@ -213,9 +213,11 @@ waiting list.
             [on-tick advance-game (/ TICKS-PER-SECOND)]
             [on-msg update-waypoint]))
 
-;; GobblerUniverse iworld? -> GobblerBundle
+;; GobblerUniverse iworld? -> GobblerUniverse
 ;; Queue the new player
 (define (queue-world uni world)
```

Comment:
if you're performing mutation i believe the convention is to put a `!` at the end of the function name